### PR TITLE
Add/email formatting for matching

### PR DESCRIPTION
**Pull request by RobertM905**

### CODE REVIEW

File: CryptoTechReminderSystem/UseCase/GetLateDevelopers.cs [hunk 1]
```diff
@@ -61,5 +61,11 @@ private static DateTimeOffset GetEndingDate(DateTimeOffset currentDateTime)
             var daysToFriday = (7 + (DayOfWeek.Friday - currentDateTime.DayOfWeek)) % 7;
             return currentDateTime.AddDays(daysToFriday);
         }
+
```
Comment (Rumaysa) [inline CryptoTechReminderSystem/UseCase/GetLateDevelopers.cs]: Doest his negate the need for email matching, are we now matching by name? Maybe we should handle the `.co.uk` and `.com` exceptions by finding the index further down so we are certain that they have a madetech email?

File: CryptoTechReminderSystem/UseCase/GetLateDevelopers.cs [hunk 2]
```diff
@@ -61,5 +61,11 @@ private static DateTimeOffset GetEndingDate(DateTimeOffset currentDateTime)
...
+        private static string TrimEmail(string email)
+        {
+            var indexOfAt = email.IndexOf('@');
+            return email.Substring(0, indexOfAt);
+        }
```
Comment (C-gyorfi) [inline CryptoTechReminderSystem/UseCase/GetLateDevelopers.cs]: `public static string removeEmailDomainName(string email)
        {
            return email.Replace(".co.uk", "")
                .Replace(".com","");
        }`

Would it be safer to check for madetech email without domain name?

File: CryptoTechReminderSystem/UseCase/GetLateDevelopers.cs [hunk 3]
```diff
@@ -61,5 +61,11 @@ private static DateTimeOffset GetEndingDate(DateTimeOffset currentDateTime)
             var daysToFriday = (7 + (DayOfWeek.Friday - currentDateTime.DayOfWeek)) % 7;
             return currentDateTime.AddDays(daysToFriday);
         }
+
+        private static string TrimEmail(string email)
```
Comment (C-gyorfi) [inline CryptoTechReminderSystem/UseCase/GetLateDevelopers.cs]:
```suggestion
        public static string removeEmailDomainName(string email)
        {
            return email.Replace(".co.uk", "")
                .Replace(".com","");
        }
```
Would it be safer to check for madetech email without domain name?

Comment (RobertM905) [reply CryptoTechReminderSystem/UseCase/GetLateDevelopers.cs]: Could we replace `.co.uk` with `.com` or is removing both `.com` and `.co.uk` better?

Thoughts?

Comment (C-gyorfi) [reply CryptoTechReminderSystem/UseCase/GetLateDevelopers.cs]: I guess it's the same, either could work

Comment (RobertM905) [reply CryptoTechReminderSystem/UseCase/GetLateDevelopers.cs]: Changed the function here https://github.com/chubberlisk/crypto_tech/pull/34/commits/63fbc8542ee2c52ae40c0ae424431ce25fc30921.

Cheers

Comment (RobertM905) [reply CryptoTechReminderSystem/UseCase/GetLateDevelopers.cs]: Marking this as resolved for now. 
Please review the commit https://github.com/chubberlisk/crypto_tech/pull/34/commits/63fbc8542ee2c52ae40c0ae424431ce25fc30921 if you have any suggestions for changes.

**Pull request merged**